models/article.ts

```diff
@@ -0,0 +1,27 @@
+export interface Writer {
+  nickname: string;
+  id: number;
+}
+
+export interface Article {
+  updatedAt: string;
+  createdAt: string;
+  likeCount: number;
+  writer: Writer;
+  image: string;
+  content: string;
+  title: string;
+  id: number;
+}
+
+export interface ArticleResponse {
+  totalCount: number;
+  list: Article[];
+}
+
+export interface ArticleQuery {
+  page: number;
+  pageSize: number;
+  orderBy: "like" | "recent";
+  keyword: string | null;
```

Comment:
p4;
keyword 쿼리의 경우 optional하게 처리하여 사용하는게 더 좋을 것 같아요!
```suggestion
  keyword?: string;
```